Index: wl18xx/net/wireless/nl80211.c
===================================================================
--- wl18xx.orig/net/wireless/nl80211.c
+++ wl18xx/net/wireless/nl80211.c
@@ -8338,7 +8338,7 @@ static int nl80211_get_coalesce(struct s
 	if (!msg)
 		return -ENOMEM;
 
-	hdr = nl80211hdr_put(msg, info->snd_portid, info->snd_seq, 0,
+	hdr = nl80211hdr_put(msg, genl_info_snd_portid(info), info->snd_seq, 0,
 			     NL80211_CMD_GET_COALESCE);
 	if (!hdr)
 		goto nla_put_failure;
